### ISSUE TITLE
[Feature Request] support for sending phone's screen capture w/audio to obs

### ISSUE THREAD

**Issue by JoqniX** [SiiliCam/SiiliCam-obs#6]
hello! firstly i like to thank you for making this app it does really helps alot. I've been using it.

so about my feature request. 
i was thinking since silicam can be used as phone cam but what if, if it could support screen capturing and stuff? and send that feed from mob to obs! so that means no need to use those external adb apps or screen mirroring apps etc?  so no more extra additional steps needed to get what you want to acheive?

for example. as you had given a example of a tournament setting in the readme

so this can be used in that same scenario where. 

* 1 phone acts as a webcam. and sends to silicam source. in obs. ( and now you can set a group name etc so it can be unique for example - [CAM] - Player 1

* 2nd phone can be used by player 2? ( think of this being a mobile game tournament or something? tbh idk software and apps ppl use for tourneys etc. but yea )
this phone captures the phone's screen. with internal audio ( if possible you can add mic audio support too. in the future? so the user can use the mic/ external mic in phone itself ( might be helpful for those competitive games etc where it requires you to have comms)

the this can be named as [Game] - Player 1 

and be sent to obs etc. and while viewing one can see the phone's screen with internal audio! 

idk if this could be possible or not? but again i havent seen any other apps do this. so i thought maybe you could try to be the first one or something. 

its just that i see alot of potentiality in this app. and this could help others looking for a better yet simple solution. 

but again. thank you for making such a wonderful solution out there for us!! it really means alot. keep up the good work! 

do lmk if this could be possible or not okie~
and if its not. then its fine. maybe you could suggest other apps which might have this ( i did research and i couldnt find much sadly ;w; ) 
but yea do lmk your thoughts on this! 
do have a good weekend ahead! 💖✨️

**Comment by simorautiainen** [comment]
If you mean that I would send the screen capture of the phone as well as the camera data? or like make a version where you could just send the screen capture and maybe the games audio? I think this would also be a cool idea and perhaps not integrate it to the SiiliCam video broadcaster, but make another application which implements your request.

**Comment by JoqniX** [comment]
wait its possible to send and use both camera and screencapture and audio too? ohh then that too would work? maybe why not make it like a toggleable feature. asking the user if they want to use the app to send all 3 data? camera, screen share & audio? or independent ones ig? 
for example. just screenshare only? or maybe even audio only? ( i use vban talkie to send my phone mic audio so if this is possible through this then that's another method for users to send phone mic audio and that to via ndi. )

camera & mic?
( since you said you think its better to make a new app instead of implementing to siilicam then ig  hmmm? ig it would be good if the new app can be have a integration to work with siilicam? for example. 
lets say the user wants camera feed & mic? then the app automatically opens siilicam app ( if its installed - ig you can use applink or deeplink ig to do that?) or if not installed. a link to google play/ios store to download. 
and set it up there ig? ) 

at first i was thinking of having all of this directly implemented in siilicam video broadcaster but ig yea it is better to have a new app instead. like you said.

**Comment by simorautiainen** [comment]
yeah maybe ill just add it to the same application, like I can send the videostream and the screen capture and the audio simultaneously yes, but I think its quite out of the scope of SiiliCam. The simpler the android app is more cleaner and easier it is to use. Like why would you even want to send audio from the phone in your use case or the camera? the camera is in so akward position that your double chin is the only thing or your chest that will be seen on the camera, and what comes for the audio, its veryvery easy to hit the microphones on your phone while you play a game. I even accidentally hit the mics when I just keep the phone in my hand legit.

**Comment by simorautiainen** [comment]
easier it would be if I did another application that just sends the screen capture and you would have second phone that would be the camera or webcam (coming soon that you can turn your webcam into siilisource in separate app)

**Comment by JoqniX** [comment]
yea that would work. ig a second app. 

about the audio right? i was more like thinking of a usecase. for example. let's say i connect my wireless mic to phone and go to the kitchen and use the phone as cam. so i can talk from the mic and viewers can see the cam feed. like that kind. but yea. for gaming. ig noone would use their phone mic while looking down on phone and almost hitting your main mic on desk 😆  but yu never know~ 

and yea i agree with you. sillicam is fine as it is i believe. its so simple and easy to use. so yea 2nd app might be option!
